src/cljs/rems/actions/add_member.cljs

```diff
@@ -0,0 +1,142 @@
+(ns rems.actions.add-member
+  (:require [re-frame.core :as rf]
+            [reagent.core :as r]
+            [rems.actions.action :refer [action-button action-form-view button-wrapper]]
+            [rems.atoms :refer [textarea]]
+            [rems.autocomplete :as autocomplete]
+            [rems.status-modal :refer [status-modal]]
+            [rems.text :refer [text]]
+            [rems.util :refer [fetch post!]]))
+
+(defn fetch-potential-members
+  [[user on-success]]
```

Comment:
why args in vector? is this a pattern elsewhere too?

Reply:
Removed

Reply:
Because if you use fx you must pass in a vector if you want multiple arguments. You can destructure either in reg-fx declaration or in the fn you refer to there.